filecoin-proofs/src/api/internal.rs

```diff
@@ -251,7 +245,7 @@ fn setup_params(
         "sector_bytes ({}) must be a multiple of 32",
         sector_bytes,
     );
-    let nodes = sector_bytes / 32;
+    let nodes = (sector_bytes / 32) as usize;
```

Comment:
With large enough sectors (> 128GiB), nodes will exceed u32 too.

Reply:
not only nodes, and also sector counts in some other files (when a repo > 1P, the sector number could > 4G), I would think it is better to do incremental changes in multiple commits, and make sure every step is correct.